tests/unit/test_feed_update_perf.py

```
@@ -0,0 +1,237 @@
+        db.create_all()
+        yield app
+        db.session.remove()
+        db.drop_all()
+
+
+@pytest.fixture
+def captured_queries():
+    queries = []
+
+    def before_cursor_execute(
+        conn, cursor, statement, parameters, context, executemany
```

Ruff (0.15.1)

[warning] Unused function argument: `conn`

(ARG001)